pom.xml

```diff
@@ -106,20 +106,25 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-gpg-plugin</artifactId>
-        <version>1.6</version>
+        <version>3.0.1</version>
         <executions>
           <execution>
             <id>sign-artifacts</id>
             <phase>verify</phase>
             <goals>
               <goal>sign</goal>
             </goals>
+            <configuration>
+              <gpgArguments>
+                <arg>--batch</arg>
```

Comment:
batch flag will help to fail fast in case the passphrase is wrong, without it we may fallback into the "enter the passphrase" suggestion phase (which requires TTY that we don't really need).